src/components/EnsOrAddress.tsx

```diff
@@ -9,10 +9,11 @@ type Props = {
 
 export const EnsOrAddress = ({ address }: Props) => {
   const { data } = useSWR(
-    `https://api.ensideas.com/ens/resolve/${encodeURIComponent(address)}`,
+    `https://api.ensideas.com/ens/resolve/${encodeURIComponent(address.toLowerCase())}`,
```

Comment:
any chance address is null or undefined? might not hurt to  address?.toLowerCase()

Reply:
`address` comes from props, which is typed! do we have any JS to worry about that may be passing in bad/untyped args?

Reply:
no js to worry about 👍
and yeah you're right, we should be good in this case. I brought it up bc we had an error before where we were calling toLowerCase() on something undefined so this gave me flashbacks lol but I think it was a special case. carry on!